src/DropdownStyles.js

```diff
@@ -62,7 +62,7 @@ const getDirectionStyles = (theme, direction) => {
       &::before {
         bottom: -8px;
         left: 9px;
-        border-top: 8px solid ${get('colors.blackfade15')(theme)}
+        border-top: 8px solid ${get('colors.blackfade15')(theme)};
```

Comment:
Nope, this actually makes it theme-able. We pass in the theme prop from the component
's props (either a custom theme or our default theme), then it needs to be passed into the styles here since the get function needs to be composed with a theme, otherwise it returns undefined.